package-lock.json

```diff
@@ -17,18 +17,16 @@
         },
         "node_modules/@aashutoshrathi/word-wrap": {
             "version": "1.2.6",
-            "resolved": "https://registry.npmjs.org/@aashutoshrathi/word-wrap/-/word-wrap-1.2.6.tgz",
-            "integrity": "sha512-1Yjs2SvM8TflER/OD3cOjhWWOZb58A2t7wpE2S9XfBYTiIl+XFhQG2bjy4Pu1I+EAlCNUzRDYDdFwFYUKvXcIA==",
             "dev": true,
+            "license": "MIT",
```

Comment:
I wonder why the diff for `package-lock.json` is so huge.

Could you try resetting to main and running `npm install` again?
from the `plugins` repo root:

```
git fetch
git checkout origin/main package-lock.json
npm install
```

Then commit the diff and see if it makes more sense.

Reply:
nothing changed, what npm version are you using ?

Reply:
I added some packages:
- @radix-ui/react-slider
- colorjs.io"
- ogl
- clsx
- @rollup/rollup-linux-x64-gnu (build on vercel)

Reply:
Hmm weird, adding packages is fine but I think the diff is touching many unrelated packages. Using npm `10.8.2`

Maybe you should reset `package-lock.json` from upstream?

```
git remote add upstream git@github.com:framer/plugins.git
git fetch upstream

git checkout upstream/main package-lock.json
npm install
```

Reply:
is it better now ?

Reply:
Yes 🙏  thanks!